en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,201 @@
+   :::
+
+1. Take one arm and insert the rectangle extrusion inside the rectangular hollow on the bottom plate.
+
+   [![Assembly16](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly16.png)](https://youtu.be/GOTqmjq9_3s)
+
+1. While inserting the top plate on top of this the 3 piece assembly (bottom plate, top plate & arms) have to screwed using Socket Cap Screw M3*38 and Flange Locknut M3. 
+1. Hold one side using the mini cross wrench provided in the developer kit.
+
+   [![Assembly17](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly17.png)](https://youtu.be/2rcNVekJQd0)
+
+1. Do not fasten any screws before all 3 motors are in place as this might make it difficult while you’re assembling the 3rd and 4th motor.
+
+   [![Assembly18](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly18.png)](https://youtu.be/SlKRuNoE_AY)
+
+### Propellers
+
+[![Assembly19](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly19.png)](https://youtu.be/yu75VkMaIyc)
+
+- The bottom plate indicates the direction of the motor.
+- The propellers that have a white/silver coating go on that respective motor with the similar coat.
+- The unlocking and locking of the propeller is indicated on the propeller itself.
+- Use the 4 propellers and insert them on the motors keeping the above 3 points in mind.
+
+The following parts can be placed as per usual.
+
+### GPS
+
+**Screw-** Locknut M3 4 pcs | Screw M3*10 4pcs
+
+
+1. Assemble the GPS by following the video.
+
+   [![Assembly20](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly20.png)](https://youtu.be/aiFxVJFjlos)
+  
+   This guide uses the GPS mount location suggested in Holybro’s guide.
+1. Screw the GPS mount’s bottom end on the payload holder side using Locknut M3 & Screw M3*10
+
+   [![Assembly21](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly21.png)](https://youtu.be/uG5UKy3FrIc)
+
+### Pixhawk 6C
+
+- The wire from the PM02 goes to POWER1 in Pixhawk
+- The telemetry goes to TELEM1
+- The GPS to GPS1
+
+[![Assembly22](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/assembly22.png)](https://youtu.be/wFlr_I3jERQ)
+
+### Companion Computer (Optional)
+
+**Screw-** Socket Cap Screw M2.5*12 4pcs | Nylon Standoff M2.5*5 4pcs Locknut M2.5 4pcs
+
+The X500 kit is provides space for a companion computer, such as Raspberry Pi or Jetson nano can be placed here [TBD].
+- Insert 4 Socket Cap Screw M2.5*12 and put the standoffs on the same.
+- Now place the companion computer and assemble it using Locknut M2.5
+
+### Camera
+
+- Cameras such as Intel Realsense depth/ tracking camera or Structure Core can be mounted using the Depth Camera Mount
+- Simply insert the mount inside the 2 bars and use the screws according to the camera you’re using.
+
+![payloads_x500v2](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/payloads_x500v2.png)
+
+
+## Install/Configure PX4
```

Comment:
I made this up based on the previous guide. However it will only be accurate for v1.13. It is not accurate for main.